README.rst

```diff
@@ -1,10 +1,10 @@
 treq: High-level Twisted HTTP Client API
 ========================================
 
-|pypi|_
-|calver|_
-|coverage|_
-|documentation|_
+|pypi|
```

Comment:
I think that this is needed...as otherwise the links are not working

I was testing with https://github.com/twisted/treq/tree/382-async-await-readme?tab=readme-ov-file

see as a playground https://rsted.info.ucl.ac.be/?theme=basic&n=8fd54317743434ab9a96c339c7862241

Reply:
Hmmm, I was trying to get rid of the underscores GitHub shows on current trunk:

![image](https://github.com/twisted/treq/assets/43662/04e346d8-1c83-412e-ba75-65a68dbd8026)

But actually, the links there don't work either. I'll see if I can get that working &mdash; I'm a bit rusty on reST.

Reply:
It looks like a bug in GitHub... I see the same problem with twisted/twisted

For twisted/towncrier we don't use the substitution ... maybe update treq to just use the image directive

```
.. image:: https://img.shields.io/pypi/v/towncrier
   :alt: PyPI release
   :target: https://pypi.org/project/towncrier/
```

Reply:
The substitution seems to work, the key is to make it a link with `:target:` as you suggested. I guess the link targets `.. _calver:` were picked up implicitly by docutils? Or maybe this never worked.